graphql-spring-boot-test/src/main/java/com/graphql/spring/boot/test/GraphQLTestSubscription.java

```diff
@@ -230,8 +261,23 @@ public GraphQLResponse awaitAndGetNextResponse(final int timeout, final boolean
    *
    * @param timeToWait the time to wait, in milliseconds
    * @return the list of responses received during that time.
+   * @see #awaitAndGetAllResponses(Duration)
+   * @deprecated since 12.0.0. Use {@link #awaitAndGetAllResponses(Duration)} instead.
    */
+  @Deprecated
   public List<GraphQLResponse> awaitAndGetAllResponses(final int timeToWait) {
+    return awaitAndGetAllResponses(Duration.ofMillis(timeToWait));
+  }
+
+  /**
+   * Waits a specified amount time and returns all responses received during that time. This method
+   * does not have any expectation regarding the number of messages. The subscription will be
+   * stopped after the time elapsed.
+   *
+   * @param timeToWait the time to wait, in durations
```

Comment:
"in durations" sounds strange again. Simply "time to wait" seems enoungh, unit of measurement is now part of the input.

Reply:
I've dealt with it.